spec/jobs/update_diaper_partner_job_spec.rb

```diff
@@ -9,46 +9,36 @@
 
       it "checks the partner status is default set to pending" do
         with_features email_active: true do
-          Sidekiq::Testing.inline! do
-            expect do
-              UpdateDiaperPartnerJob.perform_async(@partner.id)
-              @partner.reload
-            end
-            expect(@partner.status).to eq("uninvited")
+          expect do
+            UpdateDiaperPartnerJob.perform_now(@partner.id)
+            @partner.reload
           end
+          expect(@partner.status).to eq("uninvited")
         end
       end
     end
 
-    context "with a unsuccessful POST response" do
+    context "with an unsuccessful POST response" do
       before do
         @partner = create(:partner)
         response = double("Response", value: nil)
         allow(DiaperPartnerClient).to receive(:post).and_return(response)
       end
 
       it "sets the partner status to error" do
-        pending
         with_features email_active: true do
-          Sidekiq::Testing.inline! do
-            expect do
-              UpdateDiaperPartnerJob.perform_async(@partner.id)
-              @partner.reload
-            end.to change { @partner.status }.to("error")
-          end
+          expect do
+            UpdateDiaperPartnerJob.perform_now(@partner.id)
+            @partner.reload
+          end.to change { @partner.status }.to("error")
         end
       end
 
       it "posts via DiaperPartnerClient" do
         with_features email_active: true do
-          Sidekiq::Testing.inline! do
-            partner = create(:partner)
-            allow(Flipper).to receive(:enabled?) { true }
-
-            expect(DiaperPartnerClient).to receive(:post)
-
-            UpdateDiaperPartnerJob.perform_async(partner.id)
-          end
+          allow(Flipper).to receive(:enabled?) { true }
```

Comment:
Do you still need to have this here if this is in that `with_feature` block?

Reply:
possibly; I didn't check and was just doing copypasta. I'll take a peek!

Reply:
:+1: